src/components/scenes/SendScene2.tsx

```diff
@@ -0,0 +1,924 @@
+      let payeeName: string | undefined
+      const notes: string[] = []
+      const payeeFioAddresses: string[] = []
+      for (const target of spendInfo.spendTargets) {
+        const { fioAddress } = target.otherParams ?? {}
+        if (fioAddress != null) {
+          const displayAmount = div(target.nativeAmount ?? '', cryptoDisplayDenomination.multiplier, DECIMAL_PRECISION)
+          const { name } = cryptoDisplayDenomination
+          notes.push(`To ${fioAddress} <- ${displayAmount} ${name} \n`)
+          payeeFioAddresses.push(fioAddress)
+          if (payeeName == null) {
+            payeeName = fioAddress
+          } else {
+            payeeName = `Multiple FIO Addresses (${notes.length.toString()})`
+          }
+        }
+      }
+      addToFioAddressCache(account, payeeFioAddresses)
+
+      if (broadcastedTx.metadata == null) {
+        broadcastedTx.metadata = {}
+      }
+      broadcastedTx.metadata.name = payeeName
+
+      if (payeeName != null && fioSender != null) {
+        let fioNotes = `${s.strings.fragment_transaction_list_sent_prefix}${s.strings.fragment_send_from_label.toLowerCase()} ${fioSender.fioAddress}\n`
+        fioNotes += fioSender.memo ? `\n${s.strings.fio_sender_memo_label}: ${fioSender.memo}\n` : ''
+        if (notes.length > 1) {
+          fioNotes += notes.join('\n')
+        }
+        broadcastedTx.metadata.notes = `${fioNotes}\n${broadcastedTx.metadata?.notes ?? ''}`
+      }
+
+      const { name, type, id } = coreWallet
+      const {
+        currencyCode,
+        nativeAmount,
+        networkFee,
+        parentNetworkFee,
+        txid,
+        ourReceiveAddresses,
+        deviceDescription,
+        networkFeeOption,
+        requestedCustomFee,
+        feeRateUsed
+      } = signedTx
+
+      logActivity(`broadcastTx: ${account.username} -- ${name ?? 'noname'} ${type} ${id}`)
+      logActivity(`
+  currencyCode: ${currencyCode}
+  nativeAmount: ${nativeAmount}
+  txid: ${txid}
+  networkFee: ${networkFee}
+  parentNetworkFee: ${parentNetworkFee ?? ''}
+  deviceDescription: ${deviceDescription ?? ''}
+  networkFeeOption: ${networkFeeOption ?? ''}
+  requestedCustomFee: ${JSON.stringify(requestedCustomFee)}
+  feeRateUsed: ${JSON.stringify(feeRateUsed)}
+  spendTargets: ${JSON.stringify(spendInfo.spendTargets)}
+  ourReceiveAddresses: ${JSON.stringify(ourReceiveAddresses)}`)
+
+      await coreWallet.saveTx(broadcastedTx)
+
+      for (const target of spendInfo.spendTargets) {
+        // Write FIO OBT per spendTarget
+        await recordFioObtData(target, currencyCode, broadcastedTx.txid)
+      }
+
+      playSendSound().catch(error => console.log(error)) // Fail quietly
+
+      if (onDone) {
+        onDone(null, broadcastedTx)
+      } else {
+        Actions.replace('transactionDetails', {
+          edgeTransaction: broadcastedTx,
+          walletId
+        })
+      }
+      Alert.alert(s.strings.transaction_success, s.strings.transaction_success_message, [
```

Comment:
Should this be a buttons modal? The native alert looks really out of place.

Reply:
This is carried over from the original SendScene. Not trying to change design or UX for existing functionality. While I do think it is a bit out of place. not sure a button modal would be the right design. Maybe a toast or something less "invasive" but out of scope for this PR.